packages/next-server/lib/router/router.ts

```diff
@@ -151,14 +154,29 @@ export default class Router implements BaseRouter {
     window.location.reload()
   }
 
+  /**
+   * Go back in history
+   */
   back() {
     window.history.back()
   }
 
+  /**
+   * Performs a `pushState` with arguments
+   * @param url
+   * @param as
+   * @param options
```

Comment:
Are these needed since the parameters are typed?

Reply:
Ouch, I forgot to add a description to it, because it's going to show in `VSCode` hints.